src/paths.rs

```diff
@@ -81,8 +81,10 @@ mod tests {
     fn test_resolving_paths() {
         let project_path = Path::new("/tmp/foo");
         let python_version = "3.7.1";
-        let mut paths_resolver = PathsResolver::new(project_path.to_path_buf(), python_version);
-        paths_resolver.venv_outside_project = true;
+        let mut settings = Settings::default();
+        settings.venv_outside_project = true;
+        let paths_resolver =
+            PathsResolver::new(project_path.to_path_buf(), python_version, settings);
```

Comment:
Nit: `into()` or `to_owned()` could work too (I'm just bickering because it's slightly shorter!)